packages/pds/src/services/account/index.ts

```diff
@@ -145,6 +145,17 @@ export class AccountService {
     await sequenceHandleUpdate(this.db, did, handle)
   }
 
+  async updateEmail(did: string, email: string) {
+    const res = await this.db.db
+      .updateTable('user_account')
+      .set({ email })
```

Comment:
I think we may want to normalize the email to lowercase as we do elsewhere:
```suggestion
      .set({ email: email.toLowerCase() })
```

Reply:
ah good call 👍